src/main/kotlin/telegramrsvpbot/RsvpBot.kt

```diff
@@ -41,10 +42,23 @@ class RsvpBot(private val rsvpService: RsvpService, private val attendeesService
 
         val keyword: String = words[0]
 
-        if (keyword == HELP) sendMessage(message, "Available commands: \n$CREATE_LIST list_name\n" +
-                "$ADD_ATTENDEE list_name\n" +
-                "$REMOVE_ATTENDEE list_name\n" +
-                "$RSVP_LIST list_name")
+        val helpMessage: String =
+            "Type $CREATE_LIST 'event name' to create a new event\n" +
+            "Type $ADD_ATTENDEE 'attendee name' to add an attendee to the current event\n" +
```

Comment:
The parameter should be event name and it will add the user to that event based on his telegram username/name. Otherwise it wouldn't know which event user wants to be added to

Reply:
oh right, I'll update